python_modules/libraries/dagster-tableau/dagster_tableau/resources.py

```diff
@@ -82,6 +82,26 @@ def get_view(
             self._server.views.get_request(f"{self._server.views.baseurl}/{view_id}")
         )
 
+    def add_data_quality_warning_to_data_source(
+        self,
+        datasource_id: str,
+        warning_type: Optional[TSC.DQWItem.WarningType] = None,
+        message: Optional[str] = None,
+        active: Optional[bool] = None,
+        severe: Optional[bool] = None,
+    ) -> List[TSC.DQWItem]:
+        """Add a data quality warning to a data source."""
```

Comment:
nit: param docstr

Reply:
Done in [225b57f](https://github.com/dagster-io/dagster/pull/25043/commits/225b57ff8a40aabab7bc6051079acf1b55e6b09a)